pubsub/pubsub.go

```diff
@@ -22,16 +22,25 @@ import (
 
 // PubSub is the interface for message buses.
 type PubSub interface {
+	Batcher
 	Init(metadata Metadata) error
 	Features() []Feature
 	Publish(req *PublishRequest) error
 	Subscribe(ctx context.Context, req SubscribeRequest, handler Handler) error
 	Close() error
 }
 
+type Batcher interface {
```

Comment:
Comment for all public types/functions/methods

Reply:
Still WIP PR. Working on docs only.